bench

```diff
@@ -23,10 +23,10 @@ init_bench_handling();
 multi MAIN ('quickstart', Str :$stop-after = '') {
     my sub checkouts() {
         my @compilers = < perl5 nqp-moar rakudo-moar >;
-        my @extras = < nqp-moar/master rakudo-moar/nom >;
+        my @extras = < nqp-moar/master rakudo-moar/master >;
         my sub good-tag($tag) {
-               $tag ~~ /^ \d**4\.\d**2 $/ && $tag ge '2015.05'
-            || $tag ~~ /^ 'v5.'(\d+)\.(\d+) $/ && +$0 >= 18 && +$0 %% 2
+               $tag ~~ /^ \d**4\.\d**2 $/ && $tag ge '2015.12'
+            || $tag ~~ /^ 'v5.'(\d+)\.(\d+) $/ && +$0 >= 18 && +$0 %% 2 && +$1 == 0
```

Comment:
Why that second change?  Why restrict the revision to 0?

Reply:
Otherwise there are a lot of Perl versions selected by default (I saw up to v5.*.4), but at least personally I'm not terribly interested in the details of Perl's performance over time (which I suspect is going to be a lot less variable than Rakudo's), but mostly as a single (or few at most) comparison to Rakudo.

Reply:
That's fair, I can accept that.